testing/test_terminal.py

```diff
@@ -506,6 +506,33 @@ def test_three():
         )
         assert result.ret == 0
 
+    def test_deselected_with_hookwrapper(self, testdir):
```

Comment:
I don't think this is reproducing the issue, I get the same output on `master` using these files:

```
λ pytest .tmp\deselect\
======================== test session starts ========================
...
collected 3 items / 1 deselected / 2 selected

.tmp\deselect\test_1.py ..                                     [100%]

============== 2 passed, 1 deselected in 0.03 seconds ===============
```

Reply:
Yeah, needs to be a hookwrapper.

Then it fails on master:

```
E       Failed: nomatch: 'collected 3 items / 1 deselected / 2 selected'
E           and: '============================= test session starts =============================='
E           and: 'platform linux -- Python 3.7.3, pytest-4.4.2.dev116+ge3e57a75.d20190426, py-1.8.1.dev5+gd714f411.d20190405, pluggy-0.9.0'
E           and: "hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/home/daniel/Vcs/pytest/.hypothesis/examples')"
E           and: 'rootdir: /tmp/pytest-of-daniel/pytest-82/test_deselected_with_hookwrapper0'
E           and: 'plugins: xdist-1.28.0, forked-1.0.2, fastest-0.0.8, hypothesis-4.14.0, testmon-0.9.15'
E           and: 'collected 3 items'
E           and: ''
E           and: 'test_deselected_with_hookwrapper.py ..                                   [100%]'
E           and: ''
E           and: '==================== 2 passed, 1 deselected in 0.01 seconds ===================='
E           and: ''
E       remains unmatched: 'collected 3 items / 1 deselected / 2 selected'
```

Thanks for catching!